src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx

```diff
@@ -4,322 +4,150 @@ tags:
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        Compatibility with popular analytics platforms
-      </td>
-
-      <td>
-        Synthetic monitoring specifically excludes scripts for [popular analytics services](https://s3.amazonaws.com/nr-synthetics-assets/default-hostnames-blacklist/production/default-hostnames-blacklist.txt), like Google Analytics. This ensures your analytics tools continue to receive the exact same data, even with thousands of monitors checking your website each month.
-
-        You can [unblock](/docs/synthetics/new-relic-synthetics/scripting-monitors/write-scripted-browsers#unblock-service) any of the services blocked by default, or [block](/docs/synthetics/new-relic-synthetics/scripting-monitors/synthetics-scripted-browser-reference#browser-addHostnameToBlacklist) additional services.
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        REST API functions
-      </td>
-
-      <td>
-        Synthetic monitoring includes a REST API, which you can use to manage:
-
-        * [Simple monitors](/docs/apis/synthetics-rest-api/monitor-examples/manage-synthetics-monitors-via-rest-api) and [scripted monitors](/docs/apis/synthetics-rest-api/monitor-examples/manage-synthetics-monitor-scripts-rest-api)
-        * [Categories and labels for monitors](/docs/apis/synthetics-rest-api/monitor-examples/use-synthetics-label-apis)
-        * [Alert notifications](/docs/apis/synthetics-rest-api/alert-examples/manage-synthetics-alert-notifications-rest-api)
-      </td>
-    </tr>
-  </tbody>
-</table>
+<figcaption>
+Go to **[one.newrelic.com](https://one.newrelic.com) > [one.newrelic.com/synthetics-nerdlets](https://one.newrelic.com/synthetics-nerdlets)** (Choose your monitor) > (View your **Summary page**)
+</figcaption>
+
+</Step>
+
+</Steps>
+
+## Create a scripted browser monitor [#script]
+
+Scripted browser monitors are dynamic monitors that can respond to page behaviors with conditional logic. This means you can collect data on customer workflows, giving you deeper insight into the functionality of your webpages. 
+
+<Tabs stacked>
+	<TabsBar>
+		<TabsBarItem id="URL">Go to a URL</TabsBarItem>
+		<TabsBarItem id="element">Interact with a page element</TabsBarItem>
+		<TabsBarItem id="wait">Wait for a page element</TabsBarItem>
+        <TabsBarItem id="login">Log in to a website</TabsBarItem>
+
+	</TabsBar>
+
+	<TabsPages>
+		<TabsPageItem id="URL">
+        The beginning of all scripts specifies the URL the monitor should check.
+
+        ```js
+        //Visit `http://telco.nrdemo.com/`
+        $webDriver.get("http://telco.nrdemo.com/");  
+        ```
+        </TabsPageItem>
+		<TabsPageItem id="element">
+        Once your monitor goes to a URL, it scans for then interacts with the page's available elements.
```

Comment:
it scans for and interacts with (or it scans for, then interacts with. Whichever you prefer).

Reply:
Went with "Once your monitor goes to a URL, it interacts with the page's available elements."

Good call out on original sentence. pretty awk!